docs/issue-management.md

```diff
@@ -0,0 +1,395 @@
+
+To summarize:
+
+* `good first issue` -- issues intended for first time contributors. Members
+  should keep an eye out for these pull requests and shepherd it through our
+  processes.
+* `help wanted` -- issues that represent clearly laid out tasks that are
+  generally tractable for new contributors. The solution has already been
+  designed and requires no further discussion from the community. This labels
+  indicates we need additional contributors to help move this task along.
+
+## Issue and PR Triage Process
+
+When new issues or PRs are created, the maintainers must triage the issue
+to ensure the information is valid, complete, and properly categorized and
+prioritized.
+
+### Issue Triage
+
+An issue is triaged in the following way:
+
+1. Ensure the issue is not a duplicate. Do a quick search against existing
+   issues to determine if the issue has been or is currently being worked on. If
+   you suspect the issue is a duplicate, apply the [`triage/duplicate`](#triage) label.
+2. Ensure that the issue has captured all the information required for the given
+   issue [`kind/<kind>`](#issue-kinds). If information or context is needed, apply the
+   `triage/needs-information`.
+3. Apply any missing [`area/<area>`](#area) labels. An issue can relate to more
+   than one area.
+4. Apply a [`priority/<priority>`](#priority) label. This may require further
+   discussion during the community meeting if the priority cannot be determined.
+   If undetermined, do not apply a priority. Issues with unassigned priorities
+   will be selected for review.
+5. Apply a [`size/<size>`](#size) label if known. This may require further
+   discussion, a research spike or review by the assigned contributor who will
+   be working on this issue. This is only an estimate of the complexity and size
+   of the issue.
+
+Once an issue has been triaged, a comment should be left for original submitter
+to respond to any applied triage labels.
+
+If all triage labels have been addressed and the issue is ready to be worked,
+apply the label `ready-to-work` so the issue can be assigned to a milestone and
+worked by a contributor.
+
+If it is determined an issue will not be resolved or not fixed, apply the
+`triage/unresolved` label and leave a reason in a comment for the original
+submitter. Unresolved issues can be closed after giving the original submitter
+an opportunity to appeal the reason supplied.
+
+### PR Triage
+
+A PR is triaged in the following way:
+
+1. Automation will ensure that the submitter has signed the [CLA](../CONTRIBUTING.md#cla).
+2. Automation will run CI tests against the submission to ensure compliance.
+3. Apply [`size/<size>`](#size) label to the submission. (TODO: we plan to
+   automate this with a GitHub action and apply size based on lines of code).
+4. Ensure that the PR references an existing issue (exceptions to this should be
+   rare). If the PR is missing this or needs any additional information, notate
```

Comment:
I thought "notate" was used specifically for music. Maybe "note it in a comment"?

Reply:
You are correct. Changed.